functests/2_performance_update/kubelet.go

```diff
@@ -0,0 +1,169 @@
+package __performance_update
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
```

Comment:
nit - move machineconfigv1 import next to other github imports

Reply:
okay addressed this in the latest patch